.evergreen/config.yml

```diff
@@ -162,16 +122,17 @@ tasks:
       - func: run tests
         vars:
           TEST_TARGET: node
-  - name: browser-tests
-    tags: ["browser"]
-    commands:
-      - func: fetch source
-        vars:
-          NODE_MAJOR_VERSION: 16
-      - func: install dependencies
-      - func: run tests
-        vars:
-          TEST_TARGET: browser
+  # TODO(NODE-3555): Karma tests pass locally still after these changes, rhel80 is just missing chrome
```

Comment:
We do not have chrome set up on rhel80 so I'm skipping these in CI (we intend to remove them) but I wanna keep the rest of the karma set up around b/c it does work locally and demonstrates the updates did not break that build